src/Transition/CreateNewIDFUsingRulesV9_4_0.f90

```diff
@@ -515,6 +515,8 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                     OutArgs(3) = 'Mode06'
                   ENDIF
 
+              CASE('PYTHONPLUGIN:INSTANCE')
```

Comment:
Add a preprocessor message to anyone transitioning up their 9.3 files with Python Plugins.  This will help get them to see the documentation changes and how they need to add an argument for state to their functions.